src/routes/auth.js

```diff
@@ -26,4 +26,10 @@ router.put(
   controllers.changePassword,
 );
 
+router.get(
+  '/logout',
+  validator(schema.refreshAccessToken),
```

Comment:
is this the wrong schema @AbegaM ?

Reply:
Yes, it was a wrong schema but it is fixed now